wasmaudioworklet/app.js

```diff
@@ -4,11 +4,11 @@ import { initVisualizer } from './visualizer/80sgrid.js';
 import { initEditor } from './editorcontroller.js';
 
 customElements.define('app-javascriptmusic',
-  class extends HTMLElement {
+  class JavaScriptMusic extends HTMLElement {
     constructor() {
       super();
       
-      const shadowRoot = this.attachShadow({mode: 'open'});
+      this.shadowRoot = this.attachShadow({mode: 'open'});
```

Comment:
`Element.shadowRoot` is nullable, but by assigning to `this.shadowRoot` in the constructor it makes the type system not require a null check further down below because it knows it is definitely not null now (plus removed the unused variable).